rating/styles2-rating.css

```diff
@@ -0,0 +1,56 @@
+/* Rating Main */
+.rating-container {
+  margin: 3rem;
+  height: 30rem;
+}
+
+.rating-main-heading {
+  color: yellow;
+}
+.rating-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;
+}
+.rating-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.rating-snippet {
+  margin: 1rem 0rem;
+}
+.standard-rating {
+  color: yellow;
```

Comment:
These give different values in different browsers
Use color variable with hex or rgb color code

Reply:
done